clients/admin-ui/cypress/e2e/privacy-requests.cy.ts

```diff
@@ -124,4 +128,43 @@ describe("Privacy Requests", () => {
         .should("have.length", 1);
     });
   });
+
+  describe("Storage Configuration", () => {
+    beforeEach(() => {
+      cy.visit("/privacy-requests/configure/storage");
```

Comment:
Could throw in a test that starts at `/privacy-requests/configure` and clicks the links to test out that page 🤷